Source/Immutable/Public/Immutable/ImmutablePassport.h

```diff
@@ -150,6 +151,175 @@ struct FImmutablePassportZkEvmGetBalanceData
     FString ToJsonString() const;
 };
 
+USTRUCT(BlueprintType)
+struct FImtblAccessListItem
+{
+    GENERATED_BODY()
+
+    UPROPERTY(BlueprintReadWrite, EditAnywhere)
+    FString address;
+
+    UPROPERTY(BlueprintReadWrite, EditAnywhere)
+    TArray<FString> storageKeys;
+
+    FString ToJsonString() const
+    {
+        FString OutString;
+        FJsonObjectConverter::UStructToJsonObjectString<FImtblAccessListItem>(*this, OutString, 0, 0, 0, nullptr, false);
+        return OutString;
+    }
```

Comment:
Can use this instead of redefining this `ToJsonString()` for most objects. https://github.com/immutable/unreal-immutable-sdk/blob/ce74f1ebb53b2471832178542823cc70570dacb2/Source/Immutable/Public/Immutable/ImmutablePassport.h#L34